clients/ts/signalr/tests/Utils.ts

```diff
@@ -3,6 +3,16 @@
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
 
+export function unhandledRejectionHandler(): void {
+    process.on("unhandledRejection", (error) => {
+        if (error && error.stack) {
```

Comment:
This `if` is unnecessary, just do `console.error(error)`. The output is the same either way:

```
> console.log(new Error("boo"))
Error: boo
    at repl:1:13
    at Script.runInThisContext (vm.js:91:20)
    at REPLServer.defaultEval (repl.js:321:29)
    at bound (domain.js:396:14)
    at REPLServer.runBound [as eval] (domain.js:409:12)
    at REPLServer.onLine (repl.js:619:10)
    at REPLServer.emit (events.js:187:15)
    at REPLServer.EventEmitter.emit (domain.js:442:20)
    at REPLServer.Interface._onLine (readline.js:290:10)
    at REPLServer.Interface._line (readline.js:638:8)
undefined
> console.log(new Error("boo").stack)
Error: boo
    at repl:1:13
    at Script.runInThisContext (vm.js:91:20)
    at REPLServer.defaultEval (repl.js:321:29)
    at bound (domain.js:396:14)
    at REPLServer.runBound [as eval] (domain.js:409:12)
    at REPLServer.onLine (repl.js:619:10)
    at REPLServer.emit (events.js:187:15)
    at REPLServer.EventEmitter.emit (domain.js:442:20)
    at REPLServer.Interface._onLine (readline.js:290:10)
    at REPLServer.Interface._line (readline.js:638:8)
undefined
>
```

Reply:
When running in a test with the .stack one I get:
```
console.error signalr\tests\Utils.ts:9
  Error: test
      at C:\Github\SignalR\clients\ts\signalr\tests\HttpConnection.test.ts:39:20
      at new Promise (<anonymous>)
      at Object.<anonymous> (C:\Github\SignalR\clients\ts\signalr\tests\HttpConnection.test.ts:38:19)
      at Object.asyncFn (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\jasmine_async.js:82:37)
      at resolve (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:52:12)
      at new Promise (<anonymous>)
      at mapper (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:39:19)
      at promise.then (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:73:82)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

And without the .stack log:
```
console.error signalr\tests\Utils.ts:11
  Error: test
      at C:\Github\SignalR\clients\ts\signalr\tests\HttpConnection.test.ts:75:20
      at new Promise (<anonymous>)
      at Object.<anonymous> (C:\Github\SignalR\clients\ts\signalr\tests\HttpConnection.test.ts:74:17)
      at Object.asyncFn (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\jasmine_async.js:82:37)
      at resolve (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:52:12)
      at new Promise (<anonymous>)
      at mapper (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:39:19)
      at promise.then (C:\Github\SignalR\clients\ts\common\node_modules\jest-jasmine2\build\queue_runner.js:73:82)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

Line 75 of `HttpConnection.test` is not where the error occurred, however line 39 is correct.